src/LibraryManager/LibraryOperationResult.cs

```diff
@@ -50,11 +50,11 @@ public LibraryInstallationResult(params IError[] error)
         /// <code>True</code> if the install was successfull; otherwise <code>False</code>.
         /// </summary>
         /// <remarks>
-        /// The value is usually <code>True</code> if the <see cref="P:Microsoft.Web.LibraryManager.Contracts.ILibraryInstallationResult.Errors" /> list is empty.
+        /// The value is usually <code>True</code> if the <see cref="P:Microsoft.Web.LibraryManager.Contracts.ILibraryOperationResult.Errors" /> list is empty.
         /// </remarks>
         public bool Success
         {
-            get { return !Cancelled && Errors.Count == 0; }
+            get { return !Cancelled && Errors.Count == 0 && !UpToDate; }
```

Comment:
This is confusing. Why is the operation only successful if it is _not_ up-to-date? Shouldn't most operations cause things to be up-to-date?

Reply:
I meant no change was necessary.... Let me rename it

Reply:
Need ideas...

Reply:
Fixed